cyences_app_for_splunk/bin/upgrade_steps.py

```diff
@@ -87,4 +103,5 @@ def upgrade_4_5_0(session_key, logger):
     ('4.3.0', upgrade_4_3_0),
     ('4.4.0', None),
     ('4.5.0', upgrade_4_5_0),
+    ('4.8.0', upgrade_4_8_0),
```

Comment:
this change we are not going to include in this release, right?

Reply:
yes. i think so